src/utils/localeText.js

```diff
@@ -53,6 +53,9 @@ const messages = {
       cycleTime: 'Cycle Time',
       deployTime: 'Deploy Time',
       title: 'Average Time per Stage',
+      cycleTimeDesc: 'The average time it takes a card from the start of the sprint untill it\'s done.',
```

Comment:
Encuentro muy verboso decir "The average time..." yo lo sacaría, pero es una cosa de opinión